packages/bits/src/lib/tabgroup/tab-heading/tab-heading.component.less

```diff
@@ -19,6 +19,12 @@
             .setCssVariable( border-bottom-color, nui-color-selected-contrast);
         }
 
+        &.active:focus-within {
+            .tab-link {
+                background: transparent;
+            }
+        }
+
```

Comment:
Is it possible to move it inside the existing active block?

For example:

```less
        &.active {
            > .tab-link {
                .setCssVariable(color, nui-color-text-default);
                .setCssVariable(background-color, nui-color-bg-content);
            }

            &:focus-within {
                .tab-link {
                    background: transparent;
                }
            }

            border-left: none;
            border-bottom: @nui-line-thick solid;
            .setCssVariable( border-bottom-color, nui-color-selected-contrast);
        }
```

Reply:
done

Reply:
thanks. ❤️